packages/reports/addon/components/common-actions/schedule.ts

```diff
@@ -100,8 +100,13 @@ export default class ScheduleActionComponent extends Component<Args> {
     if (!formats) {
       formats = defaultFormats.slice();
       const supportedFormats = featureFlag('exportFileTypes');
-      if (Array.isArray(supportedFormats)) {
+      if (Array.isArray(supportedFormats) && supportedFormats.length > 0) {
         formats = [...formats, ...supportedFormats];
+        const uniqFormats = [...new Set(formats)];
+        formats = uniqFormats.sort(function (a, b) {
+          //Sort the index based on defaultFormat inputs
+          return defaultFormats.indexOf(b) - defaultFormats.indexOf(a);
+        });
```

Comment:
maybe `uniq(formats)` would be more readable https://lodash.com/docs/3.10.1#uniq

Reply:
I'm actually find with using [...set] pattern. I like lodash, but not a fan when the functions have pretty low effort native version.